net/Socket.cpp

```diff
@@ -806,7 +806,7 @@ bool SocketPoll::insertNewUnixSocket(
 
     std::shared_ptr<StreamSocket> socket
         = StreamSocket::create<StreamSocket>(std::string(), fd, Socket::Type::Unix,
-                                             true, websocketHandler);
+                                             true, false, websocketHandler);
```

Comment:
I worry that multiple boolean arguments would be error prone.

Inline comments with the names would help, though verbose. Ideally, these would be enums.

Reply:
made this an enum now

Reply:
Thanks! Looks much better.